package.json

```diff
@@ -1,19 +1,27 @@
 {
   "name": "oceanwind",
   "version": "0.2.2",
-  "description": "compiles tailwind like shorthand syntax into css at runtime",
+  "description": "compile tailwind like shorthand syntax into css at runtime",
   "module": "index.js",
   "main": "index.js",
+  "type": "module",
```

Comment:
Nice find!

Reply:
Yeh, I was very happy to find this 😅 if you try and import a file in node without an extension then it actually suggests doing this in the error; a pretty good DX consideration.